### PR TITLE
Expand BigintRange constructor

### PR DESCRIPTION
**Pull request by rui-mo**

### CODE REVIEW

File: velox/type/Filter.h [hunk 1]
```diff
@@ -618,6 +618,27 @@ class BigintRange final : public Filter {
         upper16_(std::min<int64_t>(upper, std::numeric_limits<int16_t>::max())),
         isSingleValue_(upper_ == lower_) {}
 
+  BigintRange(
+      int64_t lower,
+      bool lowerUnbounded,
```
Comment (mbasmanova) [inline velox/type/Filter.h:623]: This change shouldn't be needed. 

lowerUnbounded can be implemented with lower set to std::numeric_limits<int64_t>::min()
lowerExclusive can be implemented by adjusting lower to lower + 1.

Ditto upperXxx.

Comment (rui-mo) [reply velox/type/Filter.h:623]: Thank you for the quick review. 
That's true. The reason of this change and https://github.com/facebookincubator/velox/pull/5815 is to create a unified API for different Ranges, so templates can be used when generating subfield filters. If they are not needed, we can change our implementations. Thanks again.

Comment (mbasmanova) [reply velox/type/Filter.h:623]: It would be nice to keep existing APIs if possible.

Comment (rui-mo) [reply velox/type/Filter.h:623]: OK, we will resolve this difference in our usage.

**Pull request closed without merging**